setup.py

```diff
@@ -338,19 +324,29 @@ def run(self):
              'test_cpu':test_cpu,
              'test_cuda':test_cuda,
              'clean' : clean,
+             'build_ext':cbuild_ext
             }
 
 extras_require = {'cuda': ['pycuda>=2015.1', 'scikit-cuda']}
 
 # do the actual work of building the package
 VERSION = get_version_info()
 
+cythonext = ['waveform.spa_tmplt']
+ext = []
+for name in cythonext:
+    e = Extension("pycbc.%s_cpu" % name,
+                  ["pycbc/%s_cpu.pyx" % name.replace('.', '/')],
+                  extra_compile_args=[ '-O3', '-w', '-msse4.2',
```

Comment:
Doesn't msse4.2 limit the available architecture? (I don't really know, this is a question not a statement). For example if we were to purchase an AMD cluster here would we have a problem with this? I think the cython modules will be shipped compiled in our distributions, so this does need to work on all supported (ie x86_64) architectures.

Thinking about this, isn't this one [large?] disadvantage for cython over weave? Weave can use `-march` as it's being compiled on the run-system.

Reply:
An example AMD machine does support see4_2, so that's apparently a non-issue, but interested in what we *couldn't* do here because of micro-architecture differences?

Reply:
So I think the highest SSE version that is *implied* by `-march=x86-64` is `SSE2`. Though in practice there may be no machines on which we would run that are so old that they do not have `SSE4.2`.